src/Microsoft.Win32.Registry/dir.props

```diff
@@ -5,6 +5,8 @@
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <IsNETCoreApp>true</IsNETCoreApp>
     <IsNETCoreAppRef>false</IsNETCoreAppRef>
+    <IsUAP>true</IsUAP>
```

Comment:
OK we can keep it for now but I suspect this will be one of the libraries flagged once we enable the PInvoke checker.